style.css

```diff
@@ -13,11 +13,11 @@ ul, li {
 body {
   margin: 0;
   font-family: sans-serif;
-  font-size: 13px;
+  font-size: 1em;
 }
 
 input {
-  font-size: 13px;
+  font-size: 1em;
```

Comment:
Same here. By the way, maybe we can have a `body, input` rule instead of 2 separate rules, to avoid repeating this font-size.